src/plone/restapi/services/content/add.py

```diff
@@ -29,6 +33,8 @@ def reply(self):
         type_ = data.get("@type", None)
         id_ = data.get("id", None)
         title = data.get("title", None)
+        translationOf = data.get("translationOf", None)
```

Comment:
@tisto crap, just realised of the naming... Poisoned by JS :) Did we agreed on the naming of parameters on the RESTAPI?

Reply:
I noticed that as well. :) We agreed not to have a naming convention and to just map over Plone var names (which are a horrible mess as we all know). Though, we should come up with a naming convention for new var names. At some point, we might even want to be able to run plone.restapi in a strict mode that uses a consistent naming. Though, that would break lots of things...

Reply:
@sneridagh I gave this some thought. 90% of our attributes are snake_case. We have some ancient lowerCamelCase attributes from AT and Zope. Though, if we ever want to become consistent in naming we will most likely force snake_case to be consistent between backend and frontend (Python and JavaScript). Therefore I'd say we make snake_case the default. I have to admit that I have a slight personal preference towards snake_case because it is a lot more readable than camelCase. It could be that the torture with Java in my early career plays a role in that as well. :)

Reply:
:) ok then! I will amend it!

Reply:
Just dropping a quick comment here. I'm curious about why forcing the consistency among python and js, which have different preferred standards in their communities (afaik snake_case is usually preferred in python, while camelCase is for sure preferred in javascript, especially in the react world). Imho, we should let js be js and keep all the camelCasing in volto (and everything else) as it is, while working on consistency in the Plone python code of course.
To add more on that, I think that if we want to make it easier to onboard js devs in the Plone world with volto, keeping it as close to the community standards should be the goal.

Reply:
@pnicolli we decided to continue to use snake_case for plone.restapi (see https://github.com/plone/plone.restapi/commit/114bb91298e5b3d06287848adb43e805dafa44bf). I guess we could convert snake_case that comes from the REST API to camelCase and be consistent in Volto. I'd love to be consistent in Plone REST API as well but that was never really an option since Plone/Zope itself is a mess and forcing a certain style in the REST API would have been even more confusing.

Reply:
@pnicolli could you please create an issue for documenting and enforcing JS naming conventions in Volto?

Reply:
Here it is, thanks to @ksuess https://github.com/plone/volto/issues/1317#issuecomment-606464898

Reply:
btw, [graphene](https://github.com/graphql-python/graphene) converts its snake case methods to camel case graphql attributes automatically.. it's a great way to integrate seamlessly to both worlds, not sure it's worth the hassle here.

Reply:
@pnicolli the problem here is that we can not expect the backend to provide snake_case and this could make the implementation brittle. In Plone we use every possible way to define vars. I am open to suggestions here or someone who wants to explore the options we have...